Automattic/wp-calypso
# Stats: Date picker - create the remaining shortcuts

Related to #81938

## Proposed Changes

* This PR begins the process of adding the remaining shortcuts, as shown here:
![image](https://github.com/Automattic/wp-calypso/assets/30754158/a94f0a3e-3ca9-4a7a-bbe4-acb26f99f356)


## Testing Instructions



* Open Calypso Live link and navigate to Stats page.
* Apply feature flag: `?flags=stats/date-control`
* Open the new dropdown and confirm the new shortcuts are visible.
* Confirm that clicking the shortcuts updates the dates in the start and end input fields.

Note: This doesn't cover any styling of the control. That will be handled in follow-up PRs. Likewise the behaviour of the "All Time" and "Custom Range" shortcuts is not yet hammered out.

## Pre-merge Checklist
